cost-analyzer/templates/NOTES.txt

```diff
@@ -2,10 +2,18 @@
 
 --------------------------------------------------
 {{- $servicePort := .Values.service.port | default 9090 -}}
-Kubecost has been successfully installed. When pods are Ready, you can enable port-forwarding with the following command:
-    
-    kubectl port-forward --namespace {{ .Release.Namespace }} deployment/{{ template "cost-analyzer.fullname" . }} {{ $servicePort }}
-    
-Next, navigate to http://localhost:{{ $servicePort }} in a web browser.
+Kubecost has been successfully installed.
+
+Please allow 5-10 minutes for Kubecost to gather metrics and report accurate health status in the UI and data to appear.
```

Comment:
Like the of this PR, but it feel like a lot of text to me. Would love to see if we can potentially be more concise. For example, it feels like this sentence could be shortened.